_examples/dart_full_stack/.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
please remove this file from the commit, it shouldn't be part of version control